python/dolfinx/wrappers/common.cpp

```diff
@@ -41,12 +41,6 @@ void common(nb::module_& m)
   m.attr("has_kahip") = dolfinx::has_kahip();
   m.attr("has_slepc") = dolfinx::has_slepc();
   m.attr("has_adios2") = dolfinx::has_adios2();
-
-#ifdef HAS_PYBIND11_SLEPC4PY
```

Comment:
I think this has been broken for a while, and is not used elsewhere. I tested on my local installation and it gave `False`, even though I surely have `slepc4py` installed.

Reply:
Furthermore, even on `main`, I can't find any trace of the macro `HAS_PYBIND11_SLEPC4PY` ever being defined.